apps/storefront/pages/[channel]/[locale]/cart.tsx

```diff
@@ -25,7 +25,11 @@ function Cart() {
 
   // we rewrite `/checkout` to the checkout domain to keep cookies
   const externalCheckoutUrl = checkout
-    ? `/checkout?checkout=${checkout.id}&locale=${currentLocale}&channel=${currentChannel.slug}`
+    ? `${process.env.NEXT_PUBLIC_CHECKOUT_APP_URL as string}/checkout-spa?checkout=${
+        checkout.id
+      }&locale=${currentLocale}&channel=${currentChannel.slug}&domain=${
+        new URL(process.env.NEXT_PUBLIC_API_URI as string).hostname
+      }`
     : "";
```

Comment:
I think we should use [url-join](https://www.npmjs.com/package/url-join) here to make it more readable

e.g.
```js
urlJoin('checkout', `?checkout=${checkout.id}`, `?locale=${currentLocale}`) // ...
```